website/docs/how-to-catalog-sync.md

```diff
@@ -0,0 +1,205 @@
+---
+sidebar_position: 1
+title: "Registering your interoperable tables across multiple catalogs"
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# Registering your interoperable tables across multiple catalogs
+
+:::danger Important
+Using Apache XTable™ (Incubating) to sync your source tables across multiple target catalogs involves running catalog sync on your
+current dataset using a bundled jar. You can create this bundled jar by following the instructions
+on the [Installation page](/docs/setup). Read through Apache XTable™'s
+[GitHub page](https://github.com/apache/incubator-xtable#building-the-project-and-running-tests) for more information.
+:::
+
+In this tutorial, we’ll show you how to use Apache XTable™ (Incubating) to enable interoperability between catalogs. 
+For example, you can expose a Hudi, Iceberg, or Delta table in Hive Metastore (HMS) and make it available in the AWS Glue Data Catalog—without manually registering each table. 
+Additionally, Apache XTable™ (Incubating) allows you to convert the table format metadata. For instance, a Delta table in HMS can be exposed as an Iceberg table in Glue.
+
+
+## Pre-requisites
+1. Source table(s) (Hudi/Delta/Iceberg) already written to your local storage or external storage locations like S3/GCS/ADLS. 
+   If you don't have the source table written in place already, you can follow the steps in this [tutorial](/docs/how-to#create-dataset) to set it up.
+2. Clone the Apache XTable™ (Incubating) [repository](https://github.com/apache/incubator-xtable) and create the
+   `xtable-utilities_2.12-0.2.0-SNAPSHOT-bundled.jar` by following the steps on the [Installation page](/docs/setup)
```

Comment:
Will this be 0.3.0 going forward?

Reply:
Yes after the release is published, docs will be updated.